src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -522,7 +522,7 @@ private Task UpdateStatusCommentAsync(IRemote darc, string prUrl, string message
             foreach (SubscriptionPullRequestUpdate update in subscriptionPullRequestUpdates)
             {
                 ISubscriptionActor actor = SubscriptionActorFactory(new ActorId(update.SubscriptionId));
-                if (!await actor.AddDependencyFlowEventAsync(flowEvent.ToString(), updateReason, "PR"))
+                if (!await actor.AddDependencyFlowEventAsync(update.BuildId, flowEvent.ToString(), updateReason, "PR"))
```

Comment:
Can you pass the enum values to AddDependencyFlowEventAsync rather than string versions? This makes mistakes more likely to be detected at compile time, and also (if we end up with a lot of these events) makes it easier to find all the references to specific events.

Reply:
The reason I didn't do that originally was two fold: I didn't want to move the Enum to Maestro.Contracts (because it's part of ISubscriptionActor), and I wasn't sure if the new dependency flow would use the same MergePolicyCheckResult enum that pull requests do, so I didn't want to bake that into the method signature in Subscription actor (and Reason is a concatenation of DependencyFlowEventReason and MergePolicyCheckResult for PRs). I'll make the change, but I wanted to explain why I had left it as strings here.